api/src/tests/transactions_test.rs

```diff
@@ -28,6 +28,13 @@ use move_core_types::{
 use rand::{distributions::Alphanumeric, thread_rng, Rng};
 use serde_json::json;
 
+#[tokio::test]
+async fn test_deserialize_genesis_transaction() {
```

Comment:
Forge rest_api test uses new REST client to get transactions, which will be deserialized into diem_api_types::Transaction.

With a page of 25 transactions, there is higher chance the client will receive genesis transaction, the deserialization failed because we can't deserialize some Move type string serialized in the Move module ABI.

So add this test to ensure we can deserialize genesis transaction.